Dockerfile

```diff
@@ -2,24 +2,16 @@
 
 
 # Build image 
-FROM node:13.5.0-alpine3.11
+FROM node:14-alpine
```

Comment:
Why not the `latest`?

Reply:
Because Node 14 is the LTS release. However, I'll update it to whatever you want.